### PR TITLE
Remove nginx references

### PR DESCRIPTION
**Pull request by joel-wright**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -71,3 +71,5 @@ CDN_DOMAIN=cdn.climatepolicyradar.org
 # Pipeline settings
 PIPELINE_BUCKET="a-bucket-for-testing-cpr-endpoints"
 
+# Default to CPR theme
+THEME=cpr
```
Comment (diversemix) [inline .env.example:75]: May as well add ROBOTS, and HOSTNAME as well

File: nginx/nginx.conf [hunk 2]
```diff
@@ -1,36 +0,0 @@
...
-        proxy_pass http://backend:8888/api/tokens;
-    }
-
-    location /health {
-        proxy_pass http://backend:8888/health;
-    }
-
-    location ~ ^/test/(.*)$ {
-        resolver 8.8.8.8;
-        proxy_pass https://httpbin.org/$1$is_args$args;
```
Comment (diversemix) [inline nginx/nginx.conf:34]: @PatFawbertMills are we good to remove this?

**Review by diversemix — CHANGES_REQUESTED** [review]
lgtm

**Review by diversemix — APPROVED** [review]
lgtm

File: .env.example (hunk 1 above)
Comment (joel-wright) [reply .env.example:75]: Will do this as a follow-up

**Pull request merged**